docs/guidelines/content-guidelines/04-ui-elements.md

```diff
@@ -0,0 +1,130 @@
+
+- Use plural for the object names, as users usually interact with multiple objects within the program, for example:
+
+  ✅ Service Instances <br>
+  ⛔️ Service Instance
+
+- Use plain, natural language. Even though some verbs may be similar in meaning, they may sound more or less natural to the reader. For example, from these pairs of verbs, the first one sounds more natural:
+
+  ✅ Choose <br>
+  ⛔️ Select
+
+  ✅ Edit <br>
+  ⛔️ Modify
+
+- Use the three-letter currency code instead of the currency symbol. This prevents ambiguity when multiple currencies share the same symbol.
+
+  ✅ USD <br>
+  ⛔️ $
+
+- Avoid ampersands (&), technical and mathematical symbols, and symbols that may have multiple meanings. Commonly used symbols are acceptable.
+
+  ✅ % CPU <br>
+  ⛔️ Due in < 5 days <br>
+  ⛔️ Save & exit
+
+
+## Labels
+
+Labels are one-line texts that appear in various types of UI elements, such as buttons, titles and headings, forms, and drop-down menus. When designing labels for the UI elements in the Kyma project, make sure to:
+
+- Use title case for titles and headings.
+- Use sentence case in drop-down menus and button labels.
+- Omit punctuation to avoid cluttering your text.
+
+### Buttons
+
+When designing a button label, you must be as straightforward and concise as possible, since you have very limited space to convey the message. Moreover, buttons are used to invite a user to perform an action. Make them as inviting and intuitive as possible. To do so:
+
+- Use action verbs that explain what will happen when a user presses the button, without requiring them to read any supporting text. Depending on your needs, you can also use plain adverb. Alternatively, add a noun to indicate what a verb or an adverb refers to. Do **not** use articles and punctuation. They only clutter your button label.
+
+  ✅ Save <br>
+  ✅ Next <br>
+  ✅ Save draft <br>
+  ✅ Next step <br>
+  ⛔️ Save a draft
+
+- Avoid labels that are too generic, such as `Yes` or `No`. They are not intuitive and require the user to read the supporting text.  
+- Use sentence case as it reads more natural and friendly to the users, which makes them more willing to press the button. Title case doesn't read as easily as sentence case and looks more unnatural.
+
+  ✅ Next section <br>
+  ⛔️ Next Section
+
+- Be careful when using words of similar meaning, such as `Delete` and `Remove`.
+
+
+## Tooltips
+
+Tooltips are messages that provide additional information about a certain UI element. Still, the text within a tooltip must contain information that brings value to the user, so avoid adding content that may be irrelevant. Tooltips cannot also contain information that is necessary to complete a given task. Such information should be visible at first glimpse. Otherwise, users will have to remember the content of a tooltip and refer back to it after the tooltip is deactivated. Here are some general guidelines on creating tooltip texts:
```

Comment:
```suggestion
Tooltips are messages that provide additional information about a certain UI element. Still, the text within a tooltip must contain information that brings value to the user, so avoid adding content that may be irrelevant. Tooltips cannot contain information that is necessary to complete a given task. Such information should be visible at first glimpse. Otherwise, users will have to remember the content of a tooltip and refer back to it after the tooltip is deactivated. Here are some general guidelines on creating tooltip texts:
```